.github/workflows/openapi-regen.yml

```diff
@@ -75,11 +92,25 @@ jobs:
         run: |
           echo "${{ steps.get-new-openapi-ver.outputs.version }}" > $OPENAPI_VER_FILE
 
+      - name: Import GPG key
+        id: import-gpg
+        uses: crazy-max/ghaction-import-gpg@v5
```

Comment:
can you check with techops/security to see if it is fine for us to use this 3rd party action?